compose-builder/add-asc-http-export-secure.yml

```diff
@@ -55,3 +55,7 @@ services:
     security_opt:
       - no-new-privileges:true
     user: "${EDGEX_USER}:${EDGEX_GROUP}"
+
+  consul:
```

Comment:
Move this to the top with `secretstore-setup`

Reply:
fixed